### DISCUSSION TITLE
Faulty Sector count error

### DISCUSSION THREAD

**Discussion by netgiga** [filecoin-project/lotus#7788] [Storage Provider] [answered]
hello. my miner info displayed Faulty Sector count
But, It does not exist in the sectors list.
Please review this as there is no solution yet.

 ------------------------------------------
Sectors:
	Total: 260
	Proving: 241
	PreCommit1: 1
	Committing: 1
	Removed: 15
	Faulty: 2
-------------------------------------------
[ajs@localhost lotus]$ lotus-miner sectors list | grep Faulty
[ajs@localhost lotus]$ lotus-miner sectors list | grep Faulty | wc -l
0
---------------------------------------

**Comment by Reiers** [comment, marked as answer]
`lotus-miner sectors list --show-removed`

Moving this to Discussions.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update for more than 7 days.
